verification/curator-service/ui/src/components/new-case-form-fields/GenomeSequences.tsx

```diff
@@ -0,0 +1,104 @@
+import { Field, FieldArray, useFormikContext } from 'formik';
+
+import AddCircleIcon from '@material-ui/icons/AddCircle';
+import Button from '@material-ui/core/Button';
+import { DateField } from './FormikFields';
+import NewCaseFormValues from './NewCaseFormValues';
+import React from 'react';
+import Scroll from 'react-scroll';
+import { TextField } from 'formik-material-ui';
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(() => ({
+    genomeSequenceSection: {
+        margin: '1em',
+    },
+    field: {
+        marginBottom: '2em',
+    },
+}));
+
+export default function GenomeSequences(): JSX.Element {
+    const { values } = useFormikContext<NewCaseFormValues>();
+    const classes = useStyles();
+    return (
+        <Scroll.Element name="genomeSequences">
+            <fieldset>
+                <legend>Genome Sequences</legend>
+                <FieldArray name="genomeSequences">
+                    {({ push }): JSX.Element => {
+                        return (
+                            <div>
+                                {values.genomeSequences &&
+                                    values.genomeSequences.map((_, index) => (
+                                        <fieldset
+                                            key={index}
```

Comment:
we shouldn't use the index as key in react lists apparently this is wrong, slows down the app and can show the wrong data :(
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Unfortunate that formik says to use the index in their example :/ https://jaredpalmer.com/formik/docs/api/fieldarray

Done.